sdk/appcenter-crashes/src/main/java/com/microsoft/appcenter/crashes/Crashes.java

```diff
@@ -555,6 +560,34 @@ private void processPendingErrors() {
 
     }
 
+    private void processPendingBreakpadErrors() {
+        post(new Runnable() {
+
+            @Override
+            public void run() {
+                for (File logFile : ErrorLogHelper.getStoredBreakpadLogFiles()) {
```

Comment:
This completely bypasses the listener system and people will not have any callbacks and ability to send a text attachment or monitor sending events. We need to integrate this in the same process as other crashes, just a different source of crash but keep all events and behaviors.

Reply:
Also I don't see modification to make hasCrashedInLastSesion/getLastSessionCrashReport work with native crash.

Reply:
I believe events and getting hasCrashedInLastSession were stretch goals outside the scope of this initial PR.

Reply:
Bu how will this look documentation wise?

Reply:
@pniko Could you just create the managed error log and then save it in the folder with the others before the normal processing takes place? You could even send the error attachments in the preprocessing steps since they're different logs and you'll already have the error id